cds_ils/static_pages/search_guide.html

```diff
@@ -0,0 +1,674 @@
+<h2 id="cds-videos">CERN library catalogue</h2>
+
+<p>This guide explains how to perform search queries on the CERN library catalogue using easy to understand examples.</p>
+
+<h3>Simple search (one or multiple terms)</h3>
+
+<p><strong>Example:</strong> <a href="#"><code>dark matter</code></a></p>
+
+<p>Results will match records with the <em>terms</em> <code>dark</code> <em>or</em> <code>matter</code> in <em>any field</em>.
+
+<p>You can require <em>presence</em> of both terms using either the <code>+</code> or <code>AND</code> operator:</p>
+
+<p><strong>Examples:</strong> <a href="#"><code>+dark +matter</code></a> or <a href="#"><code>dark AND matter</code></a></p>
+
+<p>You can require <em>absence</em> of one or more terms using either the <code>-</code> or <code>NOT</code> operator:</p>
+
+<p><strong>Examples:</strong> <a href="#"><code>-dark +matter</code></a> or <a href="#"><code>NOT dark AND matter</code></a></p>
+
+
+<h3 id="phrasesearch">Phrase search</h3>
+
+<p><strong>Example:</strong> <a href="#"><code>"dark matter"</code></a></p>
+
+<p>Results will match records with the <em>phrase</em> <code>dark matter</code> in <em>any field</em>.</p>
+
+<h3 id="fieldsearch">Field search</h3>
+
+<p><strong>Example:</strong> <a href="#"><code>title:matter</code></a></p>
+
+<p>Results will match records with the <em>term</em> <code>matter</code> in the <em>field</em> <code>title</code>. If you want to search for multiple terms in the title you must <strong>group the terms</strong> using parenthesis:</p>
+
+<p><strong>Example:</strong> <a href="#"><code>title:(dark matter)</code></a></p>
+
+<p>See the field reference below for the full list of fields you can search.</p>
+
+<h3 id="combinedsimplephraseorfieldsearch">Combined simple, phrase or field search</h3>
+
+<p><strong>Example:</strong> <a href="#"><code>+title:"dark matter" -title:experiment</code></a> or <a href="#"><code>title:(-dark +matter)</code></a></p>
+
+<p>You can combine simple, phrase and field search to construct advanced search queries.</p>
+
+<h3 id="rangesearch">Range search</h3>
+
+<p><strong>Example:</strong> <a href="#"><code>publication_date:[2020 TO 2021]</code></a> (note, you must capitalize <code>TO</code>).</p>
+
+<p>Results will match any record with a publication date between 2020-01-01 and 2021-12-31 (both dates inclusive).</p>
+
+<p>Examples of other ranges:</p>
+
+<ul>
+<li><code>date:{* TO 2020}</code> - All days until 2020.</li>
+
+<li><code>date:[2020 TO *]</code> - All days from 2020.</li>
+
+</ul>
+
+<h3 id="phrasesearch">Examples of searches on literatures</h3>
+
+<h4>Identifiers</h4>
+
+<strong>Example:</strong>
+
+<ul>
+<li><a href="#"><code>"9814749133"</code></a> or <a href="#"><code>"9789814749138"</code></a> - ISBN 10 or 13</li>
```

Comment:
There might an error here, the fields are missing (below too for DOI and ISSN):
```suggestion
<li><a href="#"><code>identifiers.scheme:"ISBN" AND identifiers.value:"9814749133"</code></a> ........
```

Reply:
This is on purpose, for 'exact' keywords I thought it was not really necessary to specify the field since it's unlikely to match anything else, but I'm not against adding it for clarity.

Regarding `identifiers.scheme:"ISBN"` however, I think there is a deeper issue: `identifiers` is an array so it's possible that the fields match different elements of the array (I'm sure there is a special DSL syntax for that though).